apps/editor/src/app/panel/subscriptionEditPanel.tsx

```diff
@@ -240,7 +241,7 @@ function SubscriptionEditPanel({id, onClose, onSave}: SubscriptionEditPanelProps
     startsAt: startsAt.toISOString(),
     paidUntil: paidUntil ? paidUntil.toISOString() : null,
     properties,
-    deactivation
+    deactivation: deactivation ? stripTypename(deactivation) : deactivation
```

Comment:
Fixes saving a subscription that is deactivated. Currently on master it throws an error

Reply:
Just didn't get around it yet :D